[2024-02-01T21:39:19.668+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: challenger_leaderboard.tf_to_df manual__2024-02-01T21:36:51.178852+00:00 [queued]>
[2024-02-01T21:39:19.674+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: challenger_leaderboard.tf_to_df manual__2024-02-01T21:36:51.178852+00:00 [queued]>
[2024-02-01T21:39:19.675+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-01T21:39:19.687+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): tf_to_df> on 2024-02-01 21:36:51.178852+00:00
[2024-02-01T21:39:19.694+0000] {standard_task_runner.py:60} INFO - Started process 85 to run task
[2024-02-01T21:39:19.695+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'challenger_leaderboard', 'tf_to_df', 'manual__2024-02-01T21:36:51.178852+00:00', '--job-id', '236', '--raw', '--subdir', 'DAGS_FOLDER/python_tft_tier.py', '--cfg-path', '/tmp/tmpmw34q4xo']
[2024-02-01T21:39:19.696+0000] {standard_task_runner.py:88} INFO - Job 236: Subtask tf_to_df
[2024-02-01T21:39:19.773+0000] {task_command.py:423} INFO - Running <TaskInstance: challenger_leaderboard.tf_to_df manual__2024-02-01T21:36:51.178852+00:00 [running]> on host 964619f82b50
[2024-02-01T21:39:19.901+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='challenger_leaderboard' AIRFLOW_CTX_TASK_ID='tf_to_df' AIRFLOW_CTX_EXECUTION_DATE='2024-02-01T21:36:51.178852+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-01T21:36:51.178852+00:00'
[2024-02-01T21:39:20.069+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-01T21:39:20.091+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=challenger_leaderboard, task_id=tf_to_df, execution_date=20240201T213651, start_date=20240201T213919, end_date=20240201T213920
[2024-02-01T21:39:20.149+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-01T21:39:20.182+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
